Main/Source/NMemory.Shared/NMemoryManage.cs

```diff
@@ -5,5 +5,7 @@ namespace NMemory
     public static class NMemoryManager
     {
 		public static bool UseDefaultForNotNullable { get; set; }
+
+        public static bool UseSqlServerTimestamp { get; set; }
```

Comment:
Do you think this should be done as a separate setting, or maybe just add a reverse? I don't know any databases other than SqlServer that use this format.
If there are no such databases, then the point of setting is only to support backward compatibility. But even in this case, there will be support for the wrong format.